src/core/Geometry.js

```diff
@@ -399,6 +399,8 @@ Object.assign( THREE.Geometry.prototype, THREE.EventDispatcher.prototype, {
 
 	computeVertexNormals: function ( areaWeighted ) {
 
+		this.computeFaceNormals();
```

Comment:
@mrdoob What is your preference? Do you want to leave `computeVertexNormals()` as is?